app/templates/components/donation/donation-container.hbs

```diff
@@ -11,19 +11,19 @@
   per month to support {{projectTitle}}.
 </p>
 
-{{#if canShowCardList}}
-  {{donation/card-list cards=cards selectedCard=defaultCard}}
-{{/if}}
 
-{{#if canAddCard}}
-  {{#donation/credit-card canSubmit=canSubmitAddCard submit=addCard canCancel=canShowCardList}}
+{{#if showCardForm}}
```

Comment:
Form is now shown initially, if there are no cards defined. Past that point we toggle between the form and the list via the "add new card" button. We still need the cancel button.